# Disable use of rc.py on Windows, which requires Google's rc.exe
# executable. Instead, use Microsoft's rc.exe to generate the
# resource files.

--- a/build/toolchain/win/tool_wrapper.py
+++ b/build/toolchain/win/tool_wrapper.py
@@ -168,6 +168,11 @@ class WinTool(object):
     """Converts .rc files to .res files."""
     env = self._GetEnv(arch)
     args = list(args)
+
+    if sys.platform == 'win32':
+      rc_exe_exit_code = subprocess.call(args, shell=True, env=env)
+      return rc_exe_exit_code
+
     rcpy_args = args[:]
     rcpy_args[0:1] = [sys.executable, os.path.join(BASE_DIR, 'rc', 'rc.py')]
     rcpy_args.append('/showIncludes')
--- a/chrome/app/chrome_dll.rc
+++ b/chrome/app/chrome_dll.rc
@@ -37,7 +37,7 @@ IDR_MAINFRAME ACCELERATORS
 BEGIN
     VK_BACK,        IDC_BACK,                   VIRTKEY
     VK_LEFT,        IDC_BACK,                   VIRTKEY, ALT
-#if BUILDFLAG(ENABLE_PRINTING)
+#if 1
     "P",            IDC_BASIC_PRINT,            VIRTKEY, CONTROL, SHIFT
 #endif
     "D",            IDC_BOOKMARK_ALL_TABS,      VIRTKEY, CONTROL, SHIFT
@@ -166,12 +166,12 @@ END
 // the icon from the current module). We can perhaps work around this in the
 // future to get the icon from the .exe, which would save a copy.
 
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if 0
 IDR_MAINFRAME       ICON                        "theme\google_chrome\win\chrome.ico"
 IDR_SXS             ICON                        "theme\google_chrome\win\chrome_sxs.ico"
 IDR_X004_DEV        ICON                        "theme\google_chrome\win\chrome_dev.ico"
 IDR_X005_BETA       ICON                        "theme\google_chrome\win\chrome_beta.ico"
-#elif BUILDFLAG(GOOGLE_CHROME_FOR_TESTING_BRANDING)
+#elif 0
 IDR_MAINFRAME       ICON                        "theme\google_chrome\google_chrome_for_testing\win\chrome.ico"
 #else
 IDR_MAINFRAME       ICON                        "theme\chromium\win\chromium.ico"
--- a/chrome/app/chrome_exe.rc
+++ b/chrome/app/chrome_exe.rc
@@ -36,7 +36,7 @@ LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_U
 // Note: chrome/installer/util/shell_util.cc depends on the order and number of
 // icons.  To avoid breaking existing shortcuts, add new icons at the end
 // (following the ordering described above).
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if 0
 IDR_MAINFRAME           ICON       "theme\\google_chrome\\win\\chrome.ico"
 // These three are no longer used, but remain as placeholders. (They cannot be
 // removed, or existing shortcuts to the later resources will break.)
@@ -46,7 +46,7 @@ IDR_MAINFRAME_4         ICON       "them
 // The SXS icon must have an index of 4, the constant is used in Chrome code to
 // identify it.
 IDR_SXS                 ICON       "theme\\google_chrome\\win\\chrome_sxs.ico"
-#elif BUILDFLAG(GOOGLE_CHROME_FOR_TESTING_BRANDING)
+#elif 0
 IDR_MAINFRAME           ICON       "theme\\google_chrome\\google_chrome_for_testing\\win\\chrome.ico"
 #else
 IDR_MAINFRAME           ICON       "theme\\chromium\\win\\chromium.ico"
@@ -57,7 +57,7 @@ IDR_MAINFRAME           ICON       "them
 // should be incremented when a new icon is added. The icon indices in
 // chrome_icon_resources_win.h should also be updated.
 
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if 0
 IDR_X001_APP_LIST       ICON       "theme\\google_chrome\\win\\app_list.ico"
 IDR_X002_APP_LIST_SXS   ICON       "theme\\google_chrome\\win\\app_list_sxs.ico"
 IDR_X003_INCOGNITO      ICON       "theme\\google_chrome\\win\\incognito.ico"
--- a/chrome/installer/setup/setup.rc
+++ b/chrome/installer/setup/setup.rc
@@ -58,7 +58,7 @@ IDI_SETUP               ICON
 #endif    // English (U.S.) resources
 /////////////////////////////////////////////////////////////////////////////
 
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if 0
 
 /////////////////////////////////////////////////////////////////////////////
 //
